### DISCUSSION TITLE
Isaac Lab Multi-Agent skrl documentation not working

### DISCUSSION THREAD

**Discussion by Saumya-Karan** [Toni-SM/skrl#237] [General]
I tried running MAPPO to train the multi-agent environment Isaac-Shadow-Hand-Over-Direct-v0 with skrl using the following command:
isaaclab.bat -p source\standalone\workflows\skrl\train.py --task Isaac-Shadow-Hand-Over-Direct-v0 --headless --algorithm MAPPO

I am getting the following error:
Error executing job with overrides: 
Traceback (most recent call last):
File “C:\Users\saumy\IsaacLabrepo\IsaacLab\source\extensions\omni.isaac.lab_tasks\omni\isaac\lab_tasks\utils\hydra.py”, line 99, in hydra_main
func(env_cfg, agent_cfg, *args, **kwargs)
File “C:\Users\saumy\IsaacLabrepo\IsaacLab\source\standalone\workflows\skrl\train.py”, line 178, in main
runner.run()
File “C:\Users\saumy\isaaclab\lib\site-packages\skrl\utils\runner\torch\runner.py”, line 376, in run
self._trainer.train()
File “C:\Users\saumy\isaaclab\lib\site-packages\skrl\trainers\torch\sequential.py”, line 84, in train
self.multi_agent_train()
File “C:\Users\saumy\isaaclab\lib\site-packages\skrl\trainers\torch\base.py”, line 293, in multi_agent_train
shared_states = self.env.state()
File “C:\Users\saumy\isaaclab\lib\site-packages\skrl\envs\wrappers\torch\isaaclab_envs.py”, line 136, in state
return self._env.state()
AttributeError: ‘OrderEnforcing’ object has no attribute ‘state’

Set the environment variable HYDRA_FULL_ERROR=1 for a complete stack trace.
2024-12-13 07:46:35 [14,426ms] [Warning] [carb] Recursive unloadAllPlugins() detected!

Please explain how to resolve.

**Comment by Toni-SM** [comment]
Hi @Saumya-Karan 

I commented on this post: https://github.com/Toni-SM/skrl/discussions/238#discussioncomment-11584334
